src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/Semantic/RazorSemanticTokensTests.cs

```diff
@@ -24,7 +24,7 @@ public class RazorSemanticTokensTests : AbstractRazorEditorTest
 
         private static readonly string s_projectPath = TestProject.GetProjectDirectory(typeof(RazorSemanticTokensTests), useCurrentDirectory: true);
 
-        protected bool GenerateBaselines { get; set; } = true;
+        protected bool GenerateBaselines { get; set; } = false;
```

Comment:
We oughtn't turn this on in checked in code since it makes SemanticTokens change the expected value.